### ISSUE TITLE
Scroll-to-top button staying hidden

### ISSUE THREAD

**Issue by Berserkir-Wolf** [squidfunk/mkdocs-material#2566]
- [ x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [ ] ... the problem doesn't occur with the default MkDocs template (N/A, material specific option)
- [X] ... the problem is not in any of my customizations (CSS, JS, template)
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

## Description

Scroll to top button only loads in 'hidden' state - but remains clickable and active.

### Expected behavior

Scroll-to-top button should be visible.

### Actual behavior

Button activates but is not visible.

### Steps to reproduce the bug

1. Enable navigation.top
2. Deploy docs

Note: Issue is there both when using mkdocs serve and when deploying via github CI.

### Package versions

* Python: 3.8.x
* MkDocs: `mkdocs --version` (uncertain, using git CI)
* Material: >`7.0.0`

### Project configuration

``` yaml
# Site Details
site_name: Ombi Docs
site_description: The official guide for Ombi 
site_author: Ombi-App
site_url: https://docs.ombi.app
edit_uri: https://github.com/Ombi-App/Ombi.Docs/edit/development/docs/
repo_url: https://github.com/Ombi-App/Ombi.Docs
repo_name: Ombi-App/Ombi.Docs

# Site appearance (logos, colours, icons)
theme:
  name: material
  language: en
  code_wrap: true
  features:
    - header.autohide
    - navigation.instant
    - navigation.tabs
    - navigation.top
  favicon: assets/logo/favicon.ico
  logo: assets/logo/ombi-logo.png
  palette:
    # Light mode
    - media: "(prefers-color-scheme: light)"
      scheme: default
      primary: indigo
      accent: indigo
      toggle:
        icon: material/toggle-switch-off-outline
        name: Switch to dark mode
    # Dark mode
    - media: "(prefers-color-scheme: dark)"
      scheme: slate
      primary: indigo
      accent: indigo
      toggle:
        icon: material/toggle-switch
        name: Switch to light mode

# Additional links
extra:
  social:
    - icon: "fontawesome/brands/discord"
      link: "https://discord.gg/Sa7wNWb"
    - icon: "fontawesome/brands/twitch"
      link: "https://www.twitch.tv/tidusjar"
    - icon: "fontawesome/brands/twitter"
      link: "https://twitter.com/intent/follow?screen_name=tidusjar"
    - icon: "fontawesome/brands/patreon"
      link: "https://patreon.com/tidusjar/Ombi"
    - icon: "fontawesome/brands/paypal"
      link: "https://paypal.me/PlexRequestsNet"

# Markdown extensions
markdown_extensions:
  - abbr
  - attr_list
  - meta
  - pymdownx.details
  - pymdownx.highlight:
      guess_lang: true
  - pymdownx.inlinehilite
  - pymdownx.keys
  - pymdownx.saneheaders
  - pymdownx.snippets
  - pymdownx.superfences
  - pymdownx.tabbed
  - sane_lists
  - toc:
      permalink: true
      toc_depth: 4

# mkdocs function extensions
plugins:
  - search
  - git-revision-date-localized:
      type: timeago
      locale: en
      fallback_to_build_date: false
  - redirects: # Pretty URLs (links with nav)
      redirect_maps:
        'settings/lidarr.md': 'settings/all-the-arrs.md' # Combined page
        'settings/radarr.md': 'settings/all-the-arrs.md' # Combined page
        'settings/sonarr.md': 'settings/all-the-arrs.md' # Combined page
        # All the 'notification' setting types
        'settings/cloudmobile.md': 'settings/notifications/cloudmobile.md'
        'settings/discord.md': 'settings/notifications/discord.md'
        'settings/email.md': 'settings/notifications/email.md'
        'settings/gotify.md': 'settings/notifications/gotify.md'
        'settings/massemail.md': 'settings/notifications/massemail.md'
        'settings/mattermost.md': 'settings/notifications/mattermost.md'
        'settings/mobile.md': 'settings/notifications/mobile.md'
        'settings/newsletter.md': 'settings/notifications/newsletter.md'
        'settings/pushbullet.md': 'settings/notifications/pushbullet.md'
        'settings/pushover.md': 'settings/notifications/pushover.md'
        'settings/slack.md': 'settings/notifications/slack.md'
        'settings/telegram.md': 'settings/notifications/telegram.md'
        'settings/twilio.md': 'settings/notifications/twilio.md'
        'settings/webhook.md': 'settings/notifications/webhook.md'
        # Guides redirects
        'discover.md': 'guides/discover.md'
        'donate.md': 'guides/donate.md'
        'installation.md': 'guides/installation.md'
        'issues.md': 'guides/issues.md'
        'preferences.md': 'guides/preferences.md'
        'requests-list.md': 'guides/requests-list.md'
        'usermanagement.md': 'guides/usermanagement.md'
        # Info redirects
        'faq.md': 'info/faq.md'
        'faults.md': 'info/known-faults.md'
        # Temporary redirect until an explanation of the update page is ready
        'settings/update.md': 'guides/updating.md'

# Navigation Layout
nav:
  - Home: index.md
  - Getting Started:
    - Installation: guides/installation.md
    - Updating: guides/updating.md
  - Settings: 
    - Generic Settings: settings/ombi.md
    - Customization: settings/customization.md
    - Landing Page: settings/landingpage.md
    - Issues: settings/issues.md
    - User Management: settings/usermanagement.md
    - Authentication: settings/authentication.md
    - Vote: settings/vote.md
    - TheMovieDB: settings/themoviedb.md
    - Plex: settings/plex.md
    - Emby: settings/emby.md
    - Jellyfin: settings/jellyfin.md
    - Sonarr: settings/all-the-arrs.md
    - DogNzb: settings/dognzb.md
    - Sickrage: settings/sickrage.md
    - Couchpotato: settings/couchpotato.md
    - DogNzb: settings/dognzb.md
    - Radarr: settings/all-the-arrs.md
    - Notifications:
      - Mobile: settings/notifications/cloudmobile.md
      - Legacy Mobile: settings/notifications/mobile.md
      - Email: settings/notifications/email.md
      - Mass Email: settings/notifications/massemail.md
      - Newsletter: settings/notifications/newsletter.md
      - Discord: settings/notifications/discord.md
      - Slack: settings/notifications/slack.md
      - Pushbullet: settings/notifications/pushbullet.md
      - Pushover: settings/notifications/pushover.md
      - Mattermost: settings/notifications/mattermost.md
      - Telegram: settings/notifications/telegram.md
      - Gotify: settings/notifications/gotify.md
      - Twilio: settings/notifications/twilio.md
      - Webhook: settings/notifications/webhook.md
    - About: settings/about.md
    - Failed Requests: settings/failedrequests.md
    - Scheduled Tasks: settings/jobs.md
    - Logs: settings/logs.md
  - Info:
    - Alternate Databases: info/alternate-databases.md
    - API Information: info/api-information.md
    - Backing Up: info/backing-up.md
    - Common Errors: info/common-errors.md
    - Common Themes: info/common-themes.md
    - Docker Containers: info/docker-containers.md
    - FAQ: info/faq.md
    - Known Faults: info/known-faults.md
    - Notification Variables: info/notification-template-variables.md
    - Ombi & Tautulli: info/ombi-tautulli.md
    - Request Flow: info/request-workflow.md
    - Reverse Proxying: info/reverse-proxy.md
    - Roles and Notifications: info/roles-and-notifications.md
    - Startup Parameters: info/startup-parameters.md
    - Suggesting Features: guides/feature-suggestions.md
    - User Roles: info/user-roles.md
  - Guides:
    - Installation: guides/installation.md
    - Migrating Databases: guides/migrating-databases.md
    - Mobile App: guides/mobile-app.md
    - Suggesting Features: guides/feature-suggestions.md
    - Updating: guides/updating.md
    - Discover: guides/discover.md
    - Donate: guides/donate.md
    - Issues: guides/issues.md
    - Preferences: guides/preferences.md
    - Requests list: guides/requests-list.md
    - User Management: guides/usermanagement.md
    - Scheduled Windows Task: guides/schedule-with-pictures.md
  - Development:
    - Building Ombi: development/building-ombi.md
    - The Docs: development/docs.md
    - Translating Ombi: development/translations.md
  - Archive:
    - Prerequisites: archive/prerequisites.md
    - V4 Specific Proxying: archive/v4-specific-proxy.md
    - Ye Olde v3 Proxy: archive/v3-specific-proxy.md
```

### System information

* Platform: Github Pages

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by squidfunk** [comment]
Thanks for reporting. One thing is that the scroll-to-top button is clickable albeit hidden, that's clearly a bug. The other thing is that the scroll-to-top button is not displayed when scrolling down - it's only displayed when scrolling up - this is intended behavior. It's to minimize accidental overlay of content, especially on small devices.

**Comment by squidfunk** [comment]
Fixed in a9b9faa3a.

**Comment by squidfunk** [comment]
Released as part of 7.1.2.

**Comment by Berserkir-Wolf** [comment]
Many thanks for that, one hell of a quick turnaround!
